src/events.ts

```diff
@@ -15,6 +15,7 @@ export enum ProjectEvents {
   ProjectSansKpsMigrationCompleted = 'project.sans_kps_migration_completed',
   ProjectConfigUpdated = 'project.config_updated',
   ProjectDatabaseUpgradeStatusChange = 'project.database_upgrade_status_change',
+  ProjectInfrastructureRestartOrResizeInitiated = 'project.infra_restart_or_resize_initiated',
```

Comment:
Drat, was too quick merging it. I was going to suggest making the event itself a `restart initiated` and include resize vs not in the data field